### PR TITLE
[RFC] Add ability to register default fields

### PR DESCRIPTION
**Pull request by mumia**

### CODE REVIEW

File: logger.go [hunk 1]
```diff
@@ -33,9 +44,15 @@ func InitWithInstance(newInstance Logger) error {
 
 	instance = newInstance
 
+	addMandatoryDefaultFields()
+
 	return nil
 }
 
+func AddDefaultField(key string, value interface{}, isContextField bool) {
```
Comment (jmdalmeida) [inline logger.go]: missing doc

File: logger_test.go [hunk 2]
```diff
@@ -57,7 +94,7 @@ func TestLogWithFieldsAndStacktrace(t *testing.T) {
 	// test stack trace strings
 	assert.Contains(t, buffer.String(), "github.com/uniplaces/go-logger.justToShowUpInStackTrace")
 	assert.Contains(t, buffer.String(), "github.com/uniplaces/go-logger.TestLogWithFieldsAndStacktrace")
-	assert.Contains(t, buffer.String(), "github.com/uniplaces/go-logger/logger_test.go:84")
+	assert.Contains(t, buffer.String(), "github.com/uniplaces/go-logger/logger_test.go:122")
```
Comment (jmdalmeida) [inline logger_test.go]: can we make this test deterministic? so it doesn't break with every change to this file

Comment (mumia) [reply logger_test.go]: Move the test method to the top of the file?

Comment (jmdalmeida) [reply logger_test.go]: maybe use a regex to check this condition: `(github.com\/uniplaces\/go-logger\/logger_test.go):[0-9]+` seems to work, and you can use assert's function `Regexp`

Comment (mumia) [reply logger_test.go]: The intention it to test where in the file the error was triggered.
This change would make it so we would only test the error happened in the test file.

**Pull request merged**